product-mini/platforms/linux-sgx/enclave-sample/Makefile

```diff
@@ -12,12 +13,11 @@ SPEC_TEST ?= 0
 # These variables are automatically set by CMakeLists.txt
 SGX_IPFS = 0
 WAMR_BUILD_GLOBAL_HEAP_POOL = 0
+WASM_BUILD_LIB_RATS = 0
```

Comment:
WAMR_BUILD_LIB_RATS, and had better put this line before L15, since `WAMR_BUILD_GLOBAL_HEAP_POOL` and `WAMR_BUILD_GLOBAL_HEAP_SIZE` are a group configurations to set the global heap.